pacemaker.spec.in

```diff
@@ -630,6 +653,7 @@ exit 0
 
 %config(noreplace) %{_sysconfdir}/sysconfig/pacemaker
 %if %{defined _unitdir}
+%ghost %{_localstatedir}/lib/rpm-state/%{name}
```

Comment:
that case should be covered by the rm I had added in %postun conditional for final remove